topics/transcriptomics/tutorials/scrna-plant/tutorial.md

```diff
@@ -0,0 +1,534 @@
+---
+layout: tutorial_hands_on
+title: "Downstream Single-cell RNA Plant analysis with ScanPy"
+subtopic: single-cell
+priority:
+
+zenodo_link: 'https://zenodo.org/record/4597857'
+tags:
+  - single-cell
+questions:
+  - Can we reclaim cell markers using a different analysis method?
+  - Are highly variable genes paramount to the analysis?
+objectives:
+  - Perform filtering, dimension reduction, and clustering
+  - Generate a DotPlot emulating the original paper using a different analysis tool
+  - Determine robust clusters across scRNA-seq pipelines
+requirements:
+  -
+    type: "internal"
+    topic_name: transcriptomics
+    tutorials:
+        - scrna-scanpy-pbmc3k
+
+time_estimation: 2H
+key_points:
+  - Filtering parameters are dependent on the dataset, and should be explored using scatter or violin plots
+  - A DotPlot is a fantastic way to validate clusters across different analyses
+contributors:
+  - mtekman
+  - beatrizserrano
+
+
+gitter: Galaxy-Training-Network/galaxy-single-cell
+
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+> ### {% icon comment %} Comment
+>
+> Please familiarise yourself with the ["Clustering 3K PBMCs with ScanPy"]({% link topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md %}) tutorial first, as much of the process is the same, and the accompanying slide deck better explains some of the methods and concepts better.
+>
+{: .comment}
+
+Single cell RNA-seq analysis is a cornerstone of developmental research and provides a great level of detail in understanding the underlying dynamic processes within tissues. In the context of plants, this highlights some of the key differentiation pathways that root cells undergo.
+
+![The various cell subpopulations that are expected within this tutorial]({% link topics/transcriptomics/images/scrna-plant/scrna-plant1.jpg %})
+
+
+This tutorial replicates the paper ["Spatiotemporal Developmental Trajectories in the Arabidopsis Root Revealed Using High-Throughput Single-Cell RNA Sequencing"](https://doi.org/10.1016/j.devcel.2019.02.022) {% cite denyer2019spatiotemporal %}, where the major plant cell types are recovered in the data as well as distinguishing between QC and meristematic cells. The original paper used the Seurat analysis suite {% cite satija2015spatial %}, but here we will use the ScanPy analysis suite {% cite wolf2018scanpy %} integrated within the single-cell resources in Galaxy {% cite tekman2020single %} .
```

Comment:
just a stylistic suggestion. Usually we put it in ()s if it just tangentially mentions a citation to discuss something else, and without () if we're talking about "The work Alice did in {% cite %} proves whatever"

```suggestion
This tutorial replicates the paper ["Spatiotemporal Developmental Trajectories in the Arabidopsis Root Revealed Using High-Throughput Single-Cell RNA Sequencing"](https://doi.org/10.1016/j.devcel.2019.02.022) ({% cite denyer2019spatiotemporal %}), where the major plant cell types are recovered in the data as well as distinguishing between QC and meristematic cells. The original paper used the Seurat analysis suite ({% cite satija2015spatial %}), but here we will use the ScanPy analysis suite ({% cite wolf2018scanpy %}) integrated within the single-cell resources in Galaxy ({% cite tekman2020single %}).
```